packages/grafana-ui/src/components/DateTimePickers/TimeZonePicker.tsx

```diff
@@ -53,6 +53,7 @@ export const TimeZonePicker: React.FC<Props> = (props) => {
 
   return (
     <Select
+      menuShouldPortal
```

Comment:
@ashharrison90 Thoughts on this, being a change in Grafana UI? I don't think this is a breaking change, but we also didn't think the original portal change wasn't either 😅 

Other Grafana UI components have added this in.

Reply:
Ahhhh I think I understand. I think this TimeZonePicker portal-ing breaks the dashboard time picker in some way that I don't _completely_ understand why.

When the TimeZonePicker portals and you select a time zone in the dashboard time range picker, the whole time range picker will close "unexpectedly". Not quite sure why this happens, but I'll pull this change out.